src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -432,6 +432,15 @@
   "com.minecolonies.coremod.gui.buildtool.hut.level": "Level %s",
   "com.minecolonies.coremod.gui.buildtool.decorations": "Decorations",
   "com.minecolonies.coremod.gui.buildtool.scans": "My schematics",
+  "gui.field.north": "Northward field size",
```

Comment:
use modid in lang keys

Reply:
Missed this before somehow. Anyways adding "com" and "coremod" is adding megabytes onto the lang files all up, and is not really current convention. So do you want "gui.minecolonies.field..." instead?

Reply:
that's up to you, ideally, it should start/contain modid to prevent collisions with everything else

Reply:
btw do not sort them, and we should be adding them to the end of the file, removing is fine, beware of project replace as it usually replaces more than you need :D

Reply:
Alright then, gui.minecolonies it is. And I never use project replace without regex, to be ultra careful :)

Reply:
Make it the default com.minecolonies.coremod.gui, those few extra bytes in exchange for consistency. Maybe we rename them in the future to a shorter one. But for now that's the full gradle modid.

Reply:
Not that gradle really has anything to do with the games I18n at all. I'll put them with the full thing then for now, but I would like to change them all in another tidy up PR if there's no big objections. It has no advantages being like this, and to me looks kinda messy. (and if someone's offering to clean up for free, wouldn't you take it? :D)

Reply:
We can clean those up when we drop 1.15 support. If you do this now, this will lead to massive conflicts with 1.16 potentially =D

Reply:
Sure, that's fair.